lib/Models/Cesium.js

```diff
@@ -639,6 +644,7 @@ function pickObject(cesium, e) {
         result.error = 'An unknown error occurred while picking features.';
     });
 
+    result.features = result.features.filter(uniqueElements);
```

Comment:
This happens too soon, right?  The list of features won't actually be populated until the promise above resolves.  Also, I'm curious why it's needed at all.  Are duplicates being added somewhere?